### PR TITLE
[TGUI] Accounts Uplink Terminal

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Converts accounts uplink terminal to TGUI

## Why It's Good For The Game
~~BLOOD FOR THE BLOOD GODS~~
I will purge nano very very soon

## Images of changes
Not logged in
image

Accounts view
image

Details view
image

Create new account
image

## Changelog
:cl: AffectedArc07
add: Accounts uplink terminal now uses TGUI
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+  const {
+    accounts,
+  } = data;
+  const [searchText, setSearchText] = useLocalState(context, "searchText", "");
+  const [sortId, _setSortId] = useLocalState(context, "sortId", "owner_name");
+  const [sortOrder, _setSortOrder] = useLocalState(context, "sortOrder", true);
+  return (
+    <Flex direction="column" height="100%">
+      <AccountsActions />
+      <Section flexGrow="1" mt="0.5rem">
```
Comment (dearmochi) [inline tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js:59]: Avoid using flex shorthands and use the `Flex.Item` component instead.

**Review by dearmochi — APPROVED** [review]
Looks good

File: tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js (hunk 1 above)
Comment (AffectedArc07) [reply tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js:59]: So put it inside the section?

Comment (dearmochi) [reply tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js:59]: The opposite, the Section inside the `Flex.Item`

Comment (AffectedArc07) [reply tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js:59]: But I want the section to fill the full height for a backdrop

Comment (dearmochi) [reply tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js:59]: Make the `Flex.Item` stretch with `grow="1"` then make the `Section` stretch as well with `height="100%"`

Comment (AffectedArc07) [reply tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js]: Done

**Pull request merged**